scripts/validation/validators.py

```diff
@@ -34,6 +34,103 @@ def run_validations(glob_path: str, validator: Callable[[str, str], Tuple[bool,
         raise ValidationError(f"Invalid files: {errors}")
 
 
+__excluded_contract = [
```

Comment:
Can you please add a comment explaining the workaround and pointing to the JIRA / Confluence page?

Reply:
yes